### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by jcpinyan**
I did some light editing to some files, so the auto-merger bot can't handle this pull request.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2208)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
That's fine. I'll still have @casey-collab take a look to check that all the links work, etc.

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

File: english/explore-outside/journey-to-flaming-tower/start-running/start-running.md [hunk 1]
```diff
@@ -6,4 +6,7 @@ Do you:
 
 [Keep running toward the tower](keep-running/keep-running.md)
 
-[Turn and confront the presence chasing after you](confront/confront.md)
\ No newline at end of file
+[Turn and confront the presence chasing after you](confront/confront.md)
+
+[Throw yourself backward to check your pursuer with your
+shoulder](shoulder-check/shoulder-check.md)
```
Comment (cbuckey-uda) [inline english/explore-outside/journey-to-flaming-tower/start-running/start-running.md]: It looks like this link is broken. I think you could fix it by making the link `../shoulder-check/shoulder-check.md`, but I'd actually prefer if you moved the `shoulder-check` directory within the `start-running` directory. Then this link should work as-is, and I think it's a better directory structure.

**Comment by cbuckey-uda** [comment]
Haha, I guess Casey refused to do anything because of all the edits. I should probably fix that.

This looks great - thank you so much for making all these fixes! If you fix the broken link, I will be happy to merge this.

**Comment by jcpinyan** [comment]
I reorganized the directories, so now the link should work properly. Thanks for your advice.

**Comment by cbuckey-uda** [comment]
You're welcome! And thanks for making the change. At this point, though, two copies of the file are showing up (in both places). Could you remove the old copy using `git rm`?

**Comment by jcpinyan** [comment]
I believe I have now removed the old copies in the wrong place. Fingers crossed.

**Comment by cbuckey-uda** [comment]
Looks good! Thanks again. :hamster:

**Pull request merged**